.coveragerc

```diff
@@ -4,7 +4,12 @@
 omit =
     */test*
     */__init__.py
```

Comment:
Also need to omit the qiita_pet/templates and qiita_pet/uimodules directories, since apparently they are also getting counted towards the test coverage even though they are HTML.